src/test/java/org/jenkinsci/plugins/workflow/steps/TimeoutStepTest.java

```diff
@@ -212,13 +216,14 @@ public void activityRestart() throws Throwable {
                         + "    echo 'NotHereYet';\n"
                         + "    sleep 10;\n"
                         + "    echo 'JustHere!';\n"
-                        + "    sleep 30;\n"
+                        + "    sleep 20;\n"
```

Comment:
A pre-commit version the main patch contained a mistake—the build did not time out until 1½× the stated period; in this case `sleep` would run for ~23s before being terminated. I was trying to strengthen the test here to verify that the 15s activity timeout really applies.